<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PosView</class>
 <widget class="QWidget" name="PosView">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="edit_barcode">
     <property name="placeholderText">
      <string>Scan barcode or type here...</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="table_cart">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
    <item>
     <widget class="QWidget" name="totalsRow">
      <layout class="QHBoxLayout" name="totalsLayout">
        <item>
         <widget class="QLabel" name="label_subtotal">
          <property name="text">
            <string>Subtotal: 0.00</string>
          </property>
          <property name="minimumSize">
            <size>
             <width>200</width>
             <height>32</height>
            </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_tax">
          <property name="text">
            <string>Tax: 0.00</string>
          </property>
          <property name="minimumSize">
            <size>
             <width>200</width>
             <height>32</height>
            </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_discount">
          <property name="text">
            <string>Discount: 0.00</string>
          </property>
          <property name="minimumSize">
            <size>
             <width>200</width>
             <height>32</height>
            </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_total">
          <property name="text">
            <string>Total: 0.00</string>
          </property>
          <property name="minimumSize">
            <size>
             <width>240</width>
             <height>36</height>
            </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_checkout">
          <property name="text">
            <string>Checkout</string>
          </property>
          <property name="minimumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
          </property>
         </widget>
        </item>
      </layout>
     </widget>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
